Makefile

```diff
@@ -286,3 +286,5 @@ uninstall:
 .SUFFIXES:
 
 -include $(wildcard *.d)
+
+$(shell mkdir -p build)
```

Comment:
The right way to do this is something like

```make
build:
	$(MKDIR) $@

build/%.o: src/%.c .flags | build
```

Reply:
Sure thing, just wanted to KISS